src/FakeItEasy/Creation/CastleDynamicProxy/CastleDynamicProxyGenerator.cs

```diff
@@ -5,13 +5,13 @@ namespace FakeItEasy.Creation.CastleDynamicProxy
     using System.Diagnostics.CodeAnalysis;
     using System.Globalization;
     using System.Linq;
+    using System.Linq.Expressions;
     using System.Reflection;
-    using System.Reflection.Emit;
     using Castle.DynamicProxy;
     using FakeItEasy.Core;
 
     internal class CastleDynamicProxyGenerator
-        : IProxyGenerator
+        : FakeItEasy.Creation.IProxyGenerator

```

Comment:
This is because Castle.Core introduced a new `IProxyGenerator` interface, which conflicts with ours